game/opengl_text.cc

```diff
@@ -262,16 +262,18 @@ void SvgTxtTheme::draw(std::vector<TZoomText> const& _text) {
 	}
 
 	double texture_ar = text_x / text_y;
-	m_texture_width = std::min(0.96, text_x/800.);
+	m_texture_width = std::min(0.96, text_x/1366); // Previous value of 800 appears to be a left-over from 800x600 times.
```

Comment:
why was this change done in the first place? only thing this does is to increase visible text size,